### ISSUE TITLE
integrate default intermediates

### ISSUE THREAD

**Issue by julian-belina** [FZJ-IEK3-VSA/glaes#67]
currently a feature in GlobEP but should be in glaes for accessibility to everyone
This issue was originally created by: @chrisjwin
